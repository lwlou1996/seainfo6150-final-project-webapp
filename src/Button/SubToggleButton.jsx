import React from 'react'

import styles from './SubToggleButton.module.css'

const SubToggleButton = (props) => {

    return(
        <button className={styles.button} onClick={props.onClick}>
            {props.bottonText}
        </button>
    );

}

export default SubToggleButton;