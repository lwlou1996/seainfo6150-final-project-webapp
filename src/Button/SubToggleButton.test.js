import React from 'react'
import {render} from "@testing-library/react";
import SubToggleButton from "./SubToggleButton";

describe('toggle button test', () => {

    it('button toggle successfully ', () => {
        const onClick = jest.fn();
        const { container } = render(<SubToggleButton buttonText={"Show more"} onClick={onClick}/>);
        expect(container).toMatchSnapshot();
    });

})