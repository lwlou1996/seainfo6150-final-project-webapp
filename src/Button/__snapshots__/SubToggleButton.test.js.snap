// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toggle button test button toggle successfully  1`] = `
<div>
  <button
    class="button"
  />
</div>
`;
